.github/workflows/pull-request-management.yml

```diff
@@ -299,13 +299,9 @@ jobs:
   ansible_test_sanity:
     name: Run ansible-test sanity validation
     runs-on: ubuntu-20.04
-    needs: [ ]
+    needs: [ pre_commit ]
     #needs: [ molecule_eos_designs, molecule_cloudvision ]
     #if: needs.cloudvision.status != 'failed' && needs.molecule_eos_designs.status != 'failed' && needs.file-changes.outputs.plugins == 'true'
-    strategy:
-      fail-fast: true
-      matrix:
-        python_version: [ "3.8", "3.9", "3.10" ]
```

Comment:
Any reason to remove support for multiple python versions? it helps us for future proofing right?

Reply:
The ansible tests themselves will be performed with ansible version 3.8, 3.9 and 3.10 since I remove the python-version argument. This was previously added as a workaround to avoid the earlier versions, since they gave some errors. Now ansible-core has the ability to set a minimum python version for tests in the config file also added in this pr.

Reply:
ok thanks